bin/process_bulk_metadata.py

```diff
@@ -247,7 +252,7 @@ def process_metadata_issues(
                             )
                             url = f"https://aclanthology.org/{anthology_id}"
                             issue.create_comment(
-                                f"The Anthology has had difficulty keeping up with the manual process we use for the large number of metadata corrections we receive. We have therefore updated our workflow with a more automatated process. We are closing this issue, and ask that you help us out by recreating your request using this new workflow. You can do this by visiting [the paper page associated with this issue]({url}) and clicking on the yellow 'Fix metadata' button. This will take you through a few steps simple steps."
+                                f"### Notice\n\nThe Anthology has had difficulty keeping up with the large number of metadata corrections we receive. We have therefore updated our workflow with a more automatated process. We are closing this issue, and ask that you help us out by recreating your request using this new workflow. You can do this by visiting [the paper page associated with this issue]({url}) and clicking on the yellow 'Fix metadata' button. This will take you through a few steps simple steps."
```

Comment:
(not sure my comment went through so trying again) This ends with "steps simple steps"

Reply:
Just remembered that, thanks!